docs/guide/models/execution.rst

```diff
@@ -0,0 +1,29 @@
+Protocols and the Execution Model
+=================================
+
+Protocols in OpenFE are built on a flexible execution model. 
+Result objects are shaped by this model, and therefore some basic
+background on it can be useful when looking into the details of simulation
+results, even if most users don't need to work with the details of the
+execution model.
```

Comment:
That last bit makes this quite a mouthful to read. Is it really necessary and/or can it be split into two sentences?